<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataTypePage</class>
 <widget class="QWizardPage" name="DataTypePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>What type of data do you have?</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>One piece of data from each study or studies with one group</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="onearm_proportion_Button">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="text">
        <string>proportion</string>
       </property>
       <property name="icon">
        <iconset resource="../images/icons.qrc">
         <normaloff>:/new_dataset/startscreens/proportion.png</normaloff>:/new_dataset/startscreens/proportion.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="onearm_mean_Button">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>mean</string>
       </property>
       <property name="icon">
        <iconset resource="../images/icons.qrc">
         <normaloff>:/new_dataset/startscreens/mean.png</normaloff>:/new_dataset/startscreens/mean.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="onearm_single_reg_coef_Button">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>65</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>regression coefficient</string>
       </property>
       <property name="icon">
        <iconset resource="../images/icons.qrc">
         <normaloff>:/new_dataset/startscreens/single_reg_coef.png</normaloff>:/new_dataset/startscreens/single_reg_coef.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="onearm_generic_effect_size_Button">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>generic
effect size</string>
       </property>
       <property name="icon">
        <iconset resource="../images/icons.qrc">
         <normaloff>:/new_dataset/startscreens/gen_eff_size.png</normaloff>:/new_dataset/startscreens/gen_eff_size.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Data on two or more groups per study</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="twoarm_proportions_Button">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>proportions</string>
       </property>
       <property name="icon">
        <iconset resource="../images/icons.qrc">
         <normaloff>:/new_dataset/startscreens/proportions.png</normaloff>:/new_dataset/startscreens/proportions.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>72</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="twoarm_means_Button">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>means</string>
       </property>
       <property name="icon">
        <iconset resource="../images/icons.qrc">
         <normaloff>:/new_dataset/startscreens/means.png</normaloff>:/new_dataset/startscreens/means.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>54</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="twoarm_smds_Button">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>SMD</string>
       </property>
       <property name="icon">
        <iconset resource="../images/icons.qrc">
         <normaloff>:/new_dataset/startscreens/smd.png</normaloff>:/new_dataset/startscreens/smd.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Data on test performance</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="diagnostic_Button">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>65</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>diagnostic</string>
       </property>
       <property name="icon">
        <iconset resource="../images/icons.qrc">
         <normaloff>:/new_dataset/startscreens/diagnostic.png</normaloff>:/new_dataset/startscreens/diagnostic.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>85</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
